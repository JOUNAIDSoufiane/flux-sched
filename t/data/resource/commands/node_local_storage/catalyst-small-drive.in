match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/catalyst-small-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/catalyst-small-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/catalyst-small-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/catalyst-small-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/catalyst-small-drive.yaml
quit
