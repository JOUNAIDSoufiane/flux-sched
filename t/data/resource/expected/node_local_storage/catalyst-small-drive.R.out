      ------------core12[1:x]
      ------------storage6[195:x]
      ---------socket1[1:s]
      ------catalyst321[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=1
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
      ------------core13[1:x]
      ------------storage7[195:x]
      ---------socket1[1:s]
      ------catalyst321[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=2
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
      ------------core14[1:x]
      ------------storage8[195:x]
      ---------socket1[1:s]
      ------catalyst321[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=3
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
      ------------core15[1:x]
      ------------storage9[195:x]
      ---------socket1[1:s]
      ------catalyst321[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=4
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
INFO: =============================
INFO: No matching resources found
INFO: JOBID=5
INFO: =============================
